app/src/main/java/space/taran/arkrate/ui/InputActivity.kt

```diff
@@ -34,13 +42,14 @@ class InputActivity(
                     exit = slideOutHorizontally()
                 ) {
                     Row(modifier.fillMaxWidth()) {
+                        var textValue by remember { mutableStateOf(if ((count[count.keys.toList()[i]]?: "-1.0").toDouble() == -1.0
```

Comment:
Would be cool to refactor this line, it's very dense and difficult to read due.
(I know it was written before, but anyway..)